### DISCUSSION TITLE
Changing scrollbar theme

### DISCUSSION THREAD

**Discussion by mde3** [saadeghi/daisyui#3088] [Q&A] [answered]
Anyone know how to disable the custom scrollbar theme that comes with "daisyui": "4.4.20" 
I just want my scrollbar to be set back to it's default

**Comment by saadeghi** [comment, marked as answer]
What is the problem? The new scrollbar color is using the current text color which adapts the current theme.

**Reply by mde3** [reply]
> What is the problem? The new scrollbar color is using the current text color which adapts the current theme.

Hey, thanks for your quick response!

I was able to solve this issue, went back into my code to debug and found a fix

## 1. The problem 
Initially, the scrollbar color was usually set as the default grey color until we updated to v4.4.20 and then got the black scrollbar color as you can see here

![before](https://github.com/saadeghi/daisyui/assets/94781787/08a96d9d-015d-4229-a09a-27a93947741b)

inside tailwind config file before adding base-content

![config before](https://github.com/saadeghi/daisyui/assets/94781787/913a6884-3077-493e-bd4b-d875dd2066cc)

## 2. The solution
Turns out by adding a base-content in our config file, you can control what color the scrollbar is set to

inside tailwind config file after adding base-content and setting the color to default gray

![config after default](https://github.com/saadeghi/daisyui/assets/94781787/98893228-5df0-4360-95a1-40ea01999709)

screenshot of after setting scrollbar color to default gray color

![after-default](https://github.com/saadeghi/daisyui/assets/94781787/e291430e-922c-483f-b903-9ca085fefd73)

## 3. Playing around

inside tailwind config file after adding base-content and setting the color to green

![config green custom](https://github.com/saadeghi/daisyui/assets/94781787/bd7c1563-385b-4169-9d8d-e264c697b07f)

screenshot of after setting scrollbar color to green color

![green](https://github.com/saadeghi/daisyui/assets/94781787/d0c1f916-1935-4859-b74e-6a8c78b382a7)

I hope this helps anyone that experienced a similar issue.

### Thanks for your response once again! @saadeghi

**Comment by saadeghi** [comment]
✅ Fixed scrollbar high contrast issue in `4.12.8`  
scrollbar color is now currentColor (default text color) with 35% transparency which is similar to OS default colors on light/dark themes.
